src/engine/sys/sdl_glimp.cpp

```diff
@@ -2595,6 +2586,24 @@ static void GLimp_InitExtensions()
 	}
 #endif
 
+	// Shader limits.
+
+	// From GL_ARB_vertex_shader.
+	glGetIntegerv( GL_MAX_VERTEX_UNIFORM_COMPONENTS_ARB, &glConfig2.maxVertexUniforms );
+
+	// From GL_ARB_vertex_program.
```

Comment:
Do we really care to note that it's not available in GL 1?

Reply:
I care that I know it is available without requiring any extension check.

Because of the other one requiring an extension, I wanted to check that no others were in the same situation and I checked them one by one. I prefer to be explicit.

Reply:
Well it would be less confusing to say required in GL 2.0 or whatever.